hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoUpdateSuite.java

```diff
@@ -180,6 +182,7 @@ private HapiSpec updateStakingFieldsWorks() {
                                 .logged());
     }
 
+    // @HapiTest recheck after TransactionRecord.getTranscationFee() is implemented
```

Comment:
I think if `calculateFees` is implemented, it will pass

Reply:
I will recheck this after #8445 completed.

Reply:
I created an issue to implement calculateFees in all handlers. I will try to open branch for all handlers